src/Components/FetchScreen/FetchScreen.tsx

```diff
@@ -73,6 +73,7 @@ const FetchScreen = () => {
         eitc: response.has_eitc ?? false,
         lifeline: response.has_lifeline ?? false,
         leap: response.has_leap ?? false,
+        Crisis_Intervention: response.has_nc_crisis_intervention ?? false,
```

Comment:
Can we rename this to make it more inline with the other benefits?
```suggestion
        crisisIntervention: response.has_nc_crisis_intervention ?? false,
```